package.json

```diff
@@ -11,8 +11,12 @@
     "prebootstrap": "node prebuild.js",
     "test": "lerna run test",
     "bootstrap": "lerna bootstrap",
-    "lint": "lerna run lint",
-    "lint:js": "lerna run lint",
+    "lint": "npm run lint:js",
```

Comment:
I'm interested in what others thing about how to set this up. We could let each package control its linting command, which is sort of how it works now. Or we can run the linter commands from the root here. I sort of prefer the former, but it's a little more duplication of commands.

Reply:
That's how I approached it originally but walked back to this after having to duplicate all of the eslint plugins into each of the package.jsons. I'm fine with either way- I sort of see linting as a repo-level concern if we're just going to use the same rules across the project.